stdlib/public/core/LazySequence.swift

```diff
@@ -68,6 +68,11 @@
 ///           return result
 ///         }
 ///       }
+///       init(nextElement: ResultElement?, base: Base, _ nextPartialResult: @escaping (ResultElement, Base.Element) -> ResultElement) {
+///         self.nextElement = nextElement
+///         self.base = base
+///         self.nextPartialResult = nextPartialResult
+///       }
 ///       private var nextElement: ResultElement? // The next result of next().
```

Comment:
Less boilerplate is better for example code, I think. It serves no didactic purpose here to make these members `private` in the example code, and it disables the synthesized initializer.

Instead of adding explicit initializers, let's delete these access modifiers to make the example compile again.

Reply:
Agreed! I think you'll need to modify the initializer calls to include all the parameter labels, too. Thanks for catching this, @GeekAndDadDad!

Reply:
Can do.  That's how I initially made it work locally so I could move forward with my exploration of LazySequences.   Just thought it'd be better to expand it to match form of "real" code in Filter.swift.

Thanks for the feedback.